app/scenes/Settings/components/ApiKeyListItem.tsx

```diff
@@ -24,6 +24,12 @@ const ApiKeyListItem = ({ apiKey, isCopied, onCopy }: Props) => {
 
   const subtitle = (
     <>
+      {apiKey.lastActiveAt && (
```

Comment:
Lets just switch these around, so it's **created ⋅ used ⋅ expiry**

Makes more logical sense